modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/data/set/view/table/DateClayTableSchemaField.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.data.set.view.table;
+
+import com.liferay.portal.kernel.json.JSONObject;
+
+/**
+ * @author Guilherme Camacho
+ */
+public class DateClayTableSchemaField extends ClayTableSchemaField {
+
+	public DateClayTableSchemaField() {
+		setContentRenderer("date");
+	}
+
+	public String getFormat() {
+		return _format;
+	}
+
+	public boolean isWithoutTime() {
+		return _withoutTime;
+	}
+
+	public void setFormat(String format) {
+		_format = format;
+	}
+
+	public void setWithoutTime(boolean withoutTime) {
+		_withoutTime = withoutTime;
+	}
```

Comment:
Hey @markocikos , I also thought in this approach before, so as you also raise the same idea, I think it's good to implement this way, so we are going to change this to remove `withoutTime` property and create `DateTimeClayTableSchemaField` extending `DateClayTableSchemaField` just keeeping the `format` property, thanks!

Reply:
Another thing you asked, just having the `format` property doesn't solve our problem, because there are a lot of possible [combinations](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat) that would be difficult creating an algorithm to predict if we should cut the time part of Date represented in ISO format.
With this new approach without `withoutTime` property we will use the contentRenderer ("date" or "datetime") to do that.